### PR TITLE
chore: use standard owlbot.py

### PR DESCRIPTION
**Pull request by suztomo**
Towards https://github.com/googleapis/google-cloud-python/issues/10920

The escaping of star is fine as long as it passes docs check.

Thank you for opening a Pull Request! Before submitting your PR, there are a few things you can do to make sure it goes smoothly:
- [x] Make sure to open an issue as a [bug/issue](https://github.com/googleapis/python-network-services/issues/new/choose) before writing your code!  That way we can discuss the change, evaluate designs, and agree on the general idea
- [x] Ensure the tests and linter pass
- [x] Code coverage does not decrease (if any source code was changed)
- [x] Appropriate docs were updated (if necessary)

Fixes #<issue_number_goes_here> 🦕

### CODE REVIEW

File: google/cloud/network_services_v1/types/grpc_route.py [hunk 1]
```diff
@@ -279,7 +279,6 @@ class Destination(proto.Message):
                   from the exact proportion defined here
                   depending on the precision an implementation
                   supports.
-
```
Comment (suztomo) [inline google/cloud/network_services_v1/types/grpc_route.py:282]: This removal of blank line failed the "docs / docs" check.

```
sphinx.errors.SphinxWarning: /home/runner/work/python-network-services/python-network-services/google/cloud/network_services_v1/types/grpc_route.py:docstring of google.cloud.network_services_v1.types.grpc_route.GrpcRoute.Destination:26:Bullet list ends without a blank line; unexpected unindent.
```

Comment (suztomo) [reply google/cloud/network_services_v1/types/grpc_route.py:282]: Googleapis-gen does have the empty line. https://github.com/googleapis/googleapis-gen/blob/master/google/cloud/networkservices/v1/networkservices-v1-py/google/cloud/network_services_v1/types/grpc_route.py

Why my environment removed the line...?

Comment (suztomo) [reply google/cloud/network_services_v1/types/grpc_route.py:282]: I was referencing old revision of googleapis-gen https://github.com/googleapis/googleapis-gen/blob/104e9c5902f64afd75bdb6507a7702eaa7b43803/google/cloud/networkservices/v1/networkservices-v1-py/google/cloud/network_services_v1/types/grpc_route.py , which does not have this empty line.

**Comment by suztomo** [comment]
"docs / docs" passed.

**Comment by suztomo** [comment]
@parthea PTAL.

**Review by parthea — APPROVED** [review]

**Pull request merged**